incubator/local-volume-provisioner/templates/pod-security-policy.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.common.podSecurityPolicy -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "local-volume-provisioner.fullname" . }}-psp
+  labels:
+		app.kubernetes.io/name: {{ include "local-volume-provisioner.name" . }}
```

Comment:
fix the spaces here

Reply:
ah, sorry about my stupid mistakes, fixed